common/index.js

```diff
@@ -86,11 +86,11 @@ function prepareRuleModel(dbresult) {
             txMin: limit.minAmount || null,
             txMax: limit.maxAmount || null,
             dailyMaxAmount: limit.maxAmountDaily || null,
-            dailyMaxCount: parseInt(limit.maxCountDaily) || null,
```

Comment:
Front-end validations do not allow non-integers, thus making this parsing redundant.